ideas/293-ulc-integration/README.md

```diff
@@ -0,0 +1,101 @@
+---
+id: 293-ulc-integration
+title: ULC integration
+status: draft
+created: 2018-08-06
+category: core
+lead-contributor: b00ris
+contributors:
+    - b00ris Go dev
+    - jeka Go dev
+exit-criteria: no
+success-metrics: no
+clear-roles: no
+future-iteration: no
+roles-needed:
+    - Go dev
+    - Go dev
+    - Clojure dev
+    - QA
+okrs:
+    - Core. [P0] LES and/or ULC are operational and used by at least 10% of all users.
+    - Research. Integrate and test ULC in a manner “graceful downgrade”: use infura by default and if it fails, start ULC node.
```

Comment:
Why is Infura used by default? Seems complex, and if anything it should be the other way around. Am I missing something?

Reply:
From my side, Infura is the lightest mode. It should be enough users, who want to use status as a chat. If you need to make transactions or communicate with smart contracts(or Infura is banned) you can switch to other options.

Reply:
@b00ris I feel like graceful downgrade is a good UX feature, but it is a bit different and I doesn't belong to this swarm. If would be trivial to implement it if ULC and LES work reliably.